src/authService.js

```diff
@@ -7,9 +7,35 @@ import {BaseConfig} from './baseConfig';
 
 @inject(Authentication, BaseConfig)
 export class AuthService {
+  /**
+   * The Authetnication instance that handles the token
```

Comment:
ah no actually i mean autoethnication, so the process of automatically adapting an ethnic identity :smile: